Migration docs for the Prisma file (#2143)
wasp-lang/wasp: web/docs/data-model/prisma-file.md

Wasp uses [Prisma](https://www.prisma.io/) to interact with the database. Prisma is a "Next-generation Node.js and TypeScript ORM" that provides a type-safe API for working with your database.

> With Prisma, you define your application's data model in a `schema.prisma` file. Read more about how are Wasp Entities and Prisma models related on the [Entities](./entities.md) page.

Reviewer: I know this is old, but this is Croatian grammar with English words:
```suggestion
With Prisma, you define your application's data model in a `schema.prisma` file. Read more about the relationship between Wasp Entities and Prisma models [Entities](./entities.md) page.
```
Or even:
```suggestion
With Prisma, you define your application's data model in a `schema.prisma` file. Read more about how Wasp Entities relate to Prisma models on the [Entities](./entities.md) page.
```
